python/docs/source/migration_guide/index.rst

```diff
@@ -20,3 +20,18 @@
 Migration Guide
```

Comment:
I think now we can remove all the contents at https://github.com/apache/spark/blob/master/docs/pyspark-migration-guide.md (like we did at https://github.com/apache/spark/blob/master/docs/sql-migration-old.md in this commit https://github.com/apache/spark/commit/7d4eb38bbcc887fb61ba7344df3f77a046ad77f8)

Reply:
The link of PySpark API documentation should be https://spark.apache.org/docs/latest/api/python/migration_guide/index.html

Reply:
Do you mean the link below? The link is pointing readers to SQL migration, not PySpark API documentation.

Reply:
Ohhh no I mean, we can remove all contents in `pyspark-migration-guide.md` but keep one line such as:

```
The migration guide is now archived on [this page](https://spark.apache.org/docs/latest/api/python/migration_guide/index.html).
```

like [sql-migration-old.md](https://github.com/apache/spark/blob/master/docs/sql-migration-old.md).

Reply:
Oh I see. Thanks.